compiler/crates/relay-lsp/src/completion/test.rs

```diff
@@ -328,7 +333,17 @@ fn inline_fragment_on_interface_with_existing_inline_fragment() {
         "#,
         None,
     );
-    assert_labels(items.unwrap(), vec!["Named", "User", "SimpleNamed"]);
+    assert_labels(
+        items.unwrap(),
+        vec![
+            "Node",
+            "User",
+            "Actor",
+            "HasJsField",
```

Comment:
If `Actor` implements `Node` this would be valid.

I had a little oversight though: For fields returning a union you can suggest all interfaces implemented by the union members. I did the same for interfaces as well. This wasn't correct though - only interfaces that implement the interface the fragment is defined for should be suggested. I've updated the code accordingly and added a test specifically for this case.